pkg/utils/utils_suite_test.go

```diff
@@ -0,0 +1,99 @@
+/*
```

Comment:
filename: you can skip "utils", it's redundant (there should be one suite per package)